src/worker/views/playerStatDists.ts

```diff
@@ -26,44 +28,61 @@ const updatePlayers = async (
 				activeSeason: inputs.season,
 			});
 		}
-
-		const stats = [
-			"gp",
-			"gs",
-			"min",
-			"fg",
-			"fga",
-			"fgp",
-			"tp",
-			"tpa",
-			"tpp",
-			"ft",
-			"fta",
-			"ftp",
-			"orb",
-			"drb",
-			"trb",
-			"ast",
-			"tov",
-			"stl",
-			"blk",
-			"pf",
-			"pts",
-			"per",
-		];
+		console.log(inputs);
+		const stats =
+			process.env.SPORT === "basketball"
```

Comment:
Same thing could be done for basketball, using PLAYER_STATS_TABLES and a dropdown... but like I said in my other comment, consider that optional, cause this PR is an improvement even without that!